sql/core/src/test/resources/sql-tests/inputs/subq01-in-subquery.sql

```diff
@@ -0,0 +1,452 @@
+-- A test suite for IN predicate subquery
```

Comment:
The reason of this renaming is to have the future test files of the same group as subqXX-<name>.sql so they are next to each other when `ls`.